test/features/deploy_travel_agency.feature

```diff
@@ -0,0 +1,44 @@
+@quarkus
+@cr
+Feature: Deploy Travel agency service and verify its functionality
+
+  Background:
+    Given Namespace is created
+    And Kogito Operator is deployed with Infinispan and Kafka operators
+    And "CR" install Kogito Data Index with 1 replicas
+    And "CR" deploy "travelapp-kogito-travel-agency.yaml" example service
+    And Kogito application "kogito-travel-agency" has 1 pods running within 10 minutes
+    And HTTP GET request on service "kogito-travel-agency" with path "travels" is successful within 1 minutes
+
+  Scenario: Travel application without required Visa
+    When Start "travels" process on service "kogito-travel-agency" with body:
+      """json
+	{
+		"traveller" : {
+			"firstName" : "John6",
+			"lastName" : "Doe",
+			"email" : "john.doe@example.com",
+			"nationality" : "American",
+			"address" : {
+				"street" : "main street",
+				"city" : "Boston",
+				"zipCode" : "10005",
+				"country" : "US"
+			}
+		},
+		"trip" : {
+			"city" : "New York",
+			"country" : "US",
+			"begin" : "2019-12-10T00:00:00.000+02:00",
+			"end" : "2019-12-15T00:00:00.000+02:00"
+		}
+	}
+      """
+    And Service "kogito-travel-agency" contains 1 instance of process with name "travels"
```

Comment:
just curiosity if we are clustering the kogito app, does it work automatically ? or should we wait a bit ?

Reply:
What do you mean by clustering the kogito app? Like having several process instances?

Reply:
yes, we have many instances of same app

Reply:
Well, I would expect the process starting to be a synchronized action (so when Kogito returns response then process flow reached "save point"). So all the process info should be stored in infinispan already.
So I would expect that informations should be available without waiting.